src/loaders/meson.build

```diff
@@ -30,6 +30,17 @@ if all_loaders or get_option('loaders').contains('jpg') == true
     endif
 endif
 
+if all_loaders or get_option('loaders').contains('webp') == true
+    if get_option('static') == true
```

Comment:
No one have a plan to complete it. I think adding static loader in this PR is not a good idea.

Reply:
I know we don't have that plan.
But, we provide loader 'all' and 'static' options in meson options.
The above combination of options can make problems since the dependency on libwebp is not strictly required for thorvg builds.
Therefore, I think it is enough to output an error message that does not support static webp.

Of course I can add a macro to external_webp and wrap the macro inside every function in external_webp.

example)
``` 
//external_webp/meson.build
 if webp_dep.found()
     config_h.set10('THORVG_WEBP_DEPENDENCY', true)
 endif
```

Wouldn't a skeleton class be better than this?

Reply:
Rather than fake-build success, User needs to notice that feature is not supported in the build-time.
probably, like this... (src/loaders/meson.build)
![image](https://github.com/thorvg/thorvg/assets/3711518/5c8e9831-1062-4808-8b24-deea395b5099)

Reply:
Making the build fail simplifies the problem.
I've updated the PR. I deleted the webp directory.